app/angular/src/server/angular-cli_config.ts

```diff
@@ -161,7 +161,7 @@ export function applyAngularCliWebpackConfig(baseConfig: any, cliWebpackConfigOp
   // todo add type for acc
   const entry = [
     ...baseConfig.entry,
-    ...Object.values(cliStyleConfig.entry).reduce((acc: any, item) => acc.concat(item), []),
+    ...Object.values<any>(cliStyleConfig.entry).reduce((acc: any, item) => acc.concat(item), []),
```

Comment:
After installing TypeScript ~3.6.0, this line errors out: `TS2461: Type 'unknown' is not an array type.`

This is because `unknown` is the type for `cliStyleConfig.entry`. This is defined as `unknown` if you don't specify so I just went ahead and updated the generic to match